src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -36,7 +37,11 @@ public QuestionDTO getQuestion(@PathVariable("id") Long id) {
     @ApiOperation(value = "Add a new question")
     @PostMapping("/")
     public QuestionDTO addQuestion(@RequestBody QuestionDTO questionDTO) {
-        Question question = questionService.createQuestion(questionConverter.convertToEntity(questionDTO));
+        Question question = questionService
+                .createQuestion(questionConverter.convertToEntity(questionDTO));
+        if (questionDTO.getAnswers() != null) {
+            questionService.addAnswers(question, questionDTO.getAnswers());
+        }
         return questionConverter.convertToDTO(question);
```

Comment:
Why do we need to convert QuestionDTO to Question and then convert the result back to DTO?